src/main/webapp/app/exam/exam-scores/exam-scores-average-scores-graph.component.ts

```diff
@@ -137,6 +163,42 @@ export class ExamScoresAverageScoresGraphComponent implements OnInit {
                     },
                 },
             },
+            // we show the pointer to indicate to the user that a data point is clickable (navigation to exercise)
+            onHover: (event: any, chartElement) => {
+                const element = chartElement[0];
+                if (element) {
+                    event.target.style.cursor = element['_index'] !== 0 ? 'pointer' : 'default';
+                }
+            },
+            // when the user clicks on a data point, we navigate to the scores page of the corresponding exercise
+            onClick: (evt) => {
+                const point: any = this.chartInstance.getElementAtEvent(evt)[0];
+
+                if (point) {
+                    const exerciseId = this.exerciseIds[point._index - 1];
+                    const exerciseType = this.exerciseTypes[point._index - 1];
+                    if (exerciseId && exerciseType) {
+                        this.navigateToExercise(exerciseId, exerciseType);
+                    }
+                }
+            },
         };
     }
+
+    /**
+     * We navigate to the exercise scores page when the user clicks on a data point
+     */
+    navigateToExercise(exerciseId: number, exerciseType: ExerciseType) {
+        this.router.navigate([
+            'course-management',
+            this.courseId,
+            'exams',
+            this.examId,
+            'exercise-groups',
+            this.averageScores.exerciseGroupId,
+            `${exerciseType}-exercises`,
+            exerciseId,
+            'scores',
+        ]);
+    }
```

Comment:
[nit] What do you think of moving this to `navigation-utils`? There are some functions similar to this: https://github.com/ls1intum/Artemis/blob/master/src/main/webapp/app/utils/navigation.utils.ts#L71-L91

Reply:
Yeah that's actually a good idea, I created a generic method in the navigation utils to navigate into subpages of exam exercises